### ISSUE TITLE
Wilderness Skulls

### ISSUE THREAD

**Issue by JonathanSirens** [JonathanSirens/Runique-Source#6]
When you enter the wilderness, the skulls that display your wilderness level are incorrect.

**Comment by high105** [comment]
Where specifically? 
I made it so you no longer are in the wilderness when being south of the edgeville wilderness ditch.
Is that what you are talking about?

**Comment by JonathanSirens** [comment]
No, its the interface that is used to show your wilderness level. It just needs to be changed to the one on Runique.
